rcon/models.py

```diff
@@ -139,13 +141,17 @@ def vip(self) -> Optional["PlayerVIP"]:
         )
 
     def get_penalty_count(self) -> PenaltyCountType:
-        penalities_type = {"KICK", "PUNISH", "TEMPBAN", "PERMABAN"}
-        counts: PenaltyCountType = {"KICK": 0, "PUNISH": 0, "TEMPBAN": 0, "PERMABAN": 0}
+        counts = defaultdict(int)
         for action in self.received_actions:
-            if action.action_type in penalities_type:
-                counts[action.action_type] += 1
+            if PlayerActionState[action.action_type] in PlayerActionState:
```

Comment:
This is a weird condition. Previously it asserted that `action.action_type` was a legitimate action type. Here it will throw a `ValueError`(?).

This should be something like `if action_state := PlayerActionState._member_map_.get(action.action_type):` instead.

Reply:
I agree, I was focused on the previous test; I changed it to a more conventional test